util/admin/admin.go

```diff
@@ -111,8 +111,10 @@ func IsJobRollbackable(job *model.Job) bool {
 		}
 	case model.ActionAddTablePartition:
 		return job.SchemaState == model.StateNone || job.SchemaState == model.StateReplicaOnly
-	case model.ActionDropColumn, model.ActionDropColumns, model.ActionModifyColumn,
-		model.ActionDropTablePartition,
+	case model.ActionModifyColumn:
```

Comment:
Could we remove this code? It seems that all states can be rollbacked to expect the public state.